cmake/configs/default.cmake

```diff
@@ -66,6 +66,7 @@ hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
 hunter_config(OpenBLAS VERSION 0.2.19-p0)
+hunter_config(OpenCL VERSION 2.1-p0)
```

Comment:
It doesn't belong here, see [README](https://github.com/ingenue/hunter). Just wrote some info: https://github.com/ingenue/hunter/pull/42#discussion_r90889722